ci/ray_ci/test_docker_container.py

```diff
@@ -35,23 +35,23 @@ def _mock_run_script(input: List[str]) -> None:
                 "rayproject/ray:123456-py38-cu118"
             )
 
-            container = RayDockerContainer("py37", "cpu", "ray-ml")
+            container = RayDockerContainer("py38", "cpu", "ray-ml")
```

Comment:
(I do not think you have to change this test, but it is okay.)

Reply:
you can replace with `py39` instead of `py38`. `py38` is the default python version so its output is a bit different.

Reply:
yea, I don't have to but I think it's nice to not have py37 in the code base :)